src/components/details-view/details-view.js

```diff
@@ -202,36 +201,36 @@ class DetailsView extends Component {
       handleExpandAll,
       listSectionId,
       onListToggle,
-      intl
+      intl,
+      history,
+      location
     } = this.props;
 
-    let { router, queryParams } = this.context;
-
     let { isSticky } = this.state;
 
     let containerClassName = cx('container', {
       locked: isSticky
     });
 
-    let historyState = router.history.location.state;
+    let historyState = history.location.state;
 
     let isListAccordionOpen = sections && sections[listSectionId];
 
     return (
       <div data-test-eholdings-details-view={type}>
         <PaneHeader
-          firstMenu={queryParams.searchType ? (
+          firstMenu={location.search ? (
```

Comment:
This is true for any `?` in the URL while the previous check was only true when you were on a search route